### ISSUE TITLE
Add North Macedonia to MK

### ISSUE THREAD

**Issue by chainsawriot** [koheiw/newsmap#55]
The name of the country has changed to North Macedonia (北マケドニア) since 2019 Feb. For English it should be fine keeping the "Macedonia" pattern. But for languages such as German (_Nordmazedonien_, [an example](https://www.tagesschau.de/ausland/europa/petkow-nordmazedonien-101.html)), the current pattern "Mazedonien" won't capture the new name.

**Comment by koheiw** [comment]
Good point. We probably should have both "North Macedonia" and "Macedonia" because many contemporary documents still use the old name.
